### PR TITLE
upower: 0.99.10 -> 0.99.11

### PR DESCRIPTION
**Pull request by dtzWill**
###### Motivation for this change

https://gitlab.freedesktop.org/upower/upower/-/tags/UPOWER_0_99_11

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by jtojnar** [comment]
Could you move the following to `nativeBuildInputs`:

    intltool
    libxslt
    docbook_xsl
    gobject-introspection

and replace `intltool` by `gettext` as suggested in the news?

**Comment by worldofpeace** [comment]
@dtzWill Did that and then some :+1:

File: nixos/modules/services/hardware/upower.nix [hunk 1]
```diff
@@ -49,55 +52,7 @@ in
...
-            RestrictRealtime = true;
-
-            # Privilege escalation
-            NoNewPrivileges = true;
-          };
-      };
-
-    system.activationScripts.upower =
-      ''
-        mkdir -m 0755 -p /var/lib/upower
```
Comment (jtojnar) [inline nixos/modules/services/hardware/upower.nix]: How does systemd handle this? I do not see `tmpfiles.d` or ~~hmm, I thought only some directives like [`WorkingDirectory`](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#WorkingDirectory=) were created but now I cannot find docs for even that.~~

Comment (worldofpeace) [reply nixos/modules/services/hardware/upower.nix]: https://gitlab.freedesktop.org/upower/upower/commit/e06bfc6a120750ed629291ddcec02f62f178fab4

**Comment by jtojnar** [comment]
Given that we hard depend on libgudev, which is glib wrapper around udev, which is part of systemd, `useSystemd = false` does not make sense anymore.

**Comment by worldofpeace** [comment]
> Given that we hard depend on libgudev, which is glib wrapper around udev, which is part of systemd, `useSystemd = false` does not make sense anymore.

It seems that polkit and pulseaudio have a `useSystemd` as well. Wonder if the situation is similar to this one.

**Review by jtojnar — APPROVED** [review]

**Comment by dtzWill** [comment]
@worldofpeace :heart_eyes: thanks, this is great! \o/

**Pull request merged**